maintainers/maintainer-list.nix

```diff
@@ -11429,6 +11429,14 @@
     name = "Khushraj Rathod";
     keys = [ { fingerprint = "1988 3FD8 EA2E B4EC 0A93  1E22 B77B 2A40 E770 2F19"; } ];
   };
+  kiara = {
+    name = "kiara";
+    handle = "kiara";
```

Comment:
This is not a valid attribute here: https://github.com/NixOS/nixpkgs/blob/50cff47c417ca9530671786e14038d39a9e08762/maintainers/maintainer-list.nix#L5-L16

The handle you want to use is the attribute name under which you put this attrset, so `kiara = { /* ... */ };` means `kiara` is already your handle.

```suggestion
```

Reply:
thanks! i pushed an update now to address this.